### PR TITLE
Evmos - Chain Integration

### PR DESCRIPTION
**Pull request by austinchandra**
PR to integrate Evmos Chain config into Keplr extension UI. Tested locally, ready for TestNet deployment.

Config Requirements:
- EVMOS_RPC_ENDPOINT (at https://cosmos.rpc.evmos.dev/)
- EVMOS_RPC_CONFIG (default)
- EVMOS_REST_ENDPOINT (at https://rest.rpc.evmos.dev/)
- EVMOS_REST_CONFIG (default)

### CODE REVIEW

File: packages/extension/src/config.ts [hunk 1]
```diff
@@ -1224,6 +1228,51 @@ export const EmbedChainInfos: ChainInfo[] = [
...
+    walletUrl:
+      process.env.NODE_ENV === "production"
+        ? "https://wallet.keplr.app/#/evmos/stake"
+        : "http://localhost:8080/#/evmos/stake",
+    walletUrlForStaking:
+      process.env.NODE_ENV === "production"
+        ? "https://wallet.keplr.app/#/evmos/stake"
+        : "http://localhost:8080/#/evmos/stake",
+    bip44: {
+      coinType: 118,
```
Comment (fedekunze) [inline packages/extension/src/config.ts:1252]: Note this uses the `eth_secp256k1` as the default key for address derivation
```suggestion
      coinType: 60,
```

File: packages/extension/src/config.ts [hunk 2]
```diff
@@ -1224,6 +1228,51 @@ export const EmbedChainInfos: ChainInfo[] = [
     },
     beta: true,
   },
+  {
+    rpc: EVMOS_RPC_ENDPOINT,
+    rpcConfig: EVMOS_RPC_CONFIG,
+    rest: EVMOS_REST_ENDPOINT,
+    restConfig: EVMOS_REST_CONFIG,
+    chainId: "evmos_9000-2",
+    chainName: "Evmos",
```
Comment (fedekunze) [inline packages/extension/src/config.ts:1237]:
```suggestion
    chainName: "Evmos Testnet",
```

File: packages/extension/src/config.ts [hunk 3]
```diff
@@ -1224,6 +1228,51 @@ export const EmbedChainInfos: ChainInfo[] = [
...
+    ],
+    feeCurrencies: [
+      {
+        coinDenom: "PHOTON",
+        coinMinimalDenom: "aphoton",
+        coinDecimals: 18,
+      },
+    ],
+    gasPriceStep: {
+      low: 0,
```
Comment (fedekunze) [inline packages/extension/src/config.ts:1270]: min fee should be 1 aphoton IMO

**Review by fedekunze — COMMENTED** [review]
Minor comments

**Comment by fedekunze** [comment]
Note: this is for the Evmos permanet testnets (i.e `evmos_9000-N` so the name of the chain should reflect that

File: packages/extension/src/config.ts [hunk 4]
```diff
@@ -1224,6 +1228,51 @@ export const EmbedChainInfos: ChainInfo[] = [
...
+        coinMinimalDenom: "aphoton",
+        coinDecimals: 18,
+      },
+    ],
+    gasPriceStep: {
+      low: 0.125 * 1e8, // 1e-6 Photon
+      average: 0.625 * 1e8, // 5e-6 Photon
+      high: 0.125 * 1e9, // 1e-5 Photons
+    },
+    features: ["stargate", "no-legacy-stdTx"],
```
Comment (fedekunze) [inline packages/extension/src/config.ts:1274]:
```suggestion
    features: ["evm", "stargate", "no-legacy-stdTx"],
    beta: true,
```

**Review by fedekunze — APPROVED** [review]
ACK

**Comment by fedekunze** [comment]
cc @Thunnini

**Comment by khoslaventures** [comment]
@austinchandra we can use `https://cosmos.rpc.evmos.dev/` to be consistent

**Comment by giansalex** [comment]
have you tried it?  keplr returns a different address https://faucet.evmos.org/
I think it is related to keyring

**Pull request closed without merging**